packages/prevent-serp-tracking/src/index.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * WhoTracks.Me
+ * https://whotracks.me/
+ *
+ * Copyright 2017-present Ghostery GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+ function safeLinkClick(event) {
+  let el = event.target;
+  while (el && !el.href) el = el.parentElement;
+
+  if (!el) return;
+
+  el.removeAttribute('ping');
+
+  const targetUrl =
+    el.pathname === '/url' && new URL(el.href).searchParams.get('url');
+
+  if (targetUrl) {
+    event.stopImmediatePropagation();
+    el.href = targetUrl;
+  }
+}
+
+document.addEventListener('click', safeLinkClick, true);
```

Comment:
what about touch events?

Reply:
I think, it was not necessary. But you can double-check on an iPhone/iPad.

Reply:
The click event is the last one, which is always triggered, on the touch devices also, just after the `touchend`, mainly for the backwards compatibility. 

I can check on iOS, however I would need to add my ipad to our provisional profile, as now I can't build it for iOS.